data/shortcodes.md

```diff
@@ -106,3 +106,4 @@
 | 0842      | holzbaukultur | Schweizer Holzbaukultur                                                                                     |               |       |
 | 0843      | woposs        | A world of possibilities. Modal pathways on the extra-long period: the diachrony of modality in the Latin language |         |       |
 | 0844      | mark16        | MARK16 Manuscripts Data                                                                                     |               |       |
+| 0845      | fagottino     | Fagottini and tenoroons: small-sized bassoons from the 18th and 19th centuries                              |               |       |
```

Comment:
@gautschr is it normal that the short name is fagottino (O at the end) and the long name starts with Fagottini (i at the end) -> no mistake, right?

Reply:
Yes, the singular (fagottino) was chosen on purpose, fagottini is simply the plural.